### PR TITLE
Task 4 get articles completed

### PR DESCRIPTION
**Pull request by parisselbygrace**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -47,4 +47,35 @@ describe("GET /api/topics", () => {
...
\ No newline at end of file
+
+  
+    describe("GET /api/articles", () => {
+        test( "return a 200: should respond with array of article objects ", () => {
+          return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then(({ body }) => {
+              expect(body.articles.length).toBe(12);
```
Comment (mezzDavies) [inline __tests__/app.test.js:58]: Good assertion and one that is not implied by other assertions (ie a forEach can only work on an array so need need to test that something is an array before that, but your checking it is not empty/of an expected length is spot on) .👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -47,4 +47,35 @@ describe("GET /api/topics", () => {
...
+              body.articles.forEach((article) => {
+                expect(article).toMatchObject({
+                  author: expect.any(String),
+                  title: expect.any(String),
+                  article_id: expect.any(Number),
+                  topic: expect.any(String),
+                  created_at: expect.any(String),
+                  votes: expect.any(Number),
+                  comment_count: expect.any(String),
```
Comment (mezzDavies) [inline __tests__/app.test.js:67]: Just a note to say you can, if you wanted - not crucial - change comment_count to be returned as a number instead of a string in your DB query (and then check it is a number not a string here).

**Review by mezzDavies — CHANGES_REQUESTED** [review]
The ticket is good but you are requiring in models and controllers using absolute paths. These need to be relative paths because once this server is hosted (on a different machine) they will no longer work. Please update this and then resubmit the PR.
